### ISSUE TITLE
Security

### ISSUE THREAD

**Issue by Lotterleben** [Lotterleben/AODVv2-Draft#12]
Justin has named "security" as one of the reasons why he didn't move AODVv2 forward. I'm not quite sure what he meant with that, though– we've resolved the forwarding vs. regeneration issue, which was a huge security problem, and [the Security Considerations thread I started ](https://www.ietf.org/mail-archive/web/manet/current/msg18846.html) (following the request of Thomas Clausen) got exactly 0 replies.
The only problem I can see is that he wasn't happy with our key management text, but as Christopher Dearlove pointed out, key management in MANETs, _especially_ for a reactive protocol, is an issue _much_ bigger than AODVv2– and one that is yet to be resolved.
Maybe the draft would need a review focussing on security aspects? Or something else? What do you think?

(In any case, we need to have a response to this point ready.)

**Comment by pv0** [comment]
Would definitely be good to get feedback on the latest draft. I imagine Justin couldn't say "security did get resolved" - we only sent the update earlier that day, so he may not have had a chance to read it, and it was too soon for any feedback from MANET to say whether they were happy.

**Comment by Lotterleben** [comment]
I agree. How do we go about getting feedback though? We could try asking Christopher if he'd be willing to share his thoughts, since he's an expert at this and he asked how we'd collect feedback when Justin's decision was announced?

**Comment by charliep51** [comment]
Hello folks,

About security:

- We have the end2end draft which will work

- I haven't yet found the recent emails about message regeneration 
versus forwarding.  I will be looking for them, but if anyone wants to 
send a pointer that would be appreciated.

- Key distribution is *not* part of AODVv2!!  We don't have to do this 
to get the document published.

Regards,
Charlie P.

**Comment by charliep51** [comment]
Hello folks,

I asked Bob Moskowitz to take a look.  I'll let you know what he says.  
I hope he's not on vacation.

Regards,
Charlie P
